context/app/static/js/components/publications/PublicationVignette/hooks.ts

```diff
@@ -0,0 +1,47 @@
+import useSWR from 'swr';
+import { useAppContext } from 'js/components/Contexts';
+import { fillUrls } from './utils';
+import { PublicationVignette } from '../types';
+
+// Fetcher function that lets SWR fetch multiple urls at once
+const multiFetcher = (...urls: string[]) => {
+  const f = (url: string) => fetch(url).then((response) => response.json());
+  return Promise.all(urls.map((url) => f(url)));
+};
```

Comment:
Would this be worth pulling out for use elsewhere?

Reply:
I'll move this to a shared utils folder so we can reuse it elsewhere, good call!

Reply:
This should be fixed now 👍